### PR TITLE
Fix comment footer overlapping over "read more" button

### LINKED ISSUE #8497: "Read More" link is covered by like button on mobile comments
This makes it very hard to press (only the corners work)

https://user-images.githubusercontent.com/34715248/148758904-4149a06e-f22c-4431-9b61-fd7db8e38922.mp4

### PR DESCRIPTION
**Pull request by notbakaneko**
closes #8497

### CODE REVIEW

**Comment by notbakaneko** [comment]
wonder if the button should be slighter larger for mobile, though 🤔

**Comment by nanaya** [comment]
I'm not sure about fixing something that only happens in dev tool so I'm waiting for more details on where else it happens.

**Review by nanaya — CHANGES_REQUESTED** [review]
looks like it's better to add margin on the `read more` button instead with even more margin for mobile view

File: resources/assets/less/bem/comment.less [hunk 1]
```diff
@@ -197,7 +197,7 @@
   }
 
   &__row {
-    margin: -2px -5px;
+    gap: 2px;
```
Comment (nanaya) [inline resources/assets/less/bem/comment.less:200]: should be 4px

File: resources/assets/less/bem/comment.less [hunk 2]
```diff
@@ -243,6 +243,10 @@
     .link-hover({
       color: @osu-colour-l1;
     });
+
+    @media @mobile {
+      padding: 5px 0;
```
Comment (nanaya) [inline resources/assets/less/bem/comment.less:248]: can use some margin for desktop as well as currently it's the only row with missing margin

**Review by nanaya — CHANGES_REQUESTED** [review]

**Review by nanaya — APPROVED** [review]

**Pull request merged**